loadtest/user/userentity/actions.go

```diff
@@ -291,6 +291,19 @@ func (ue *UserEntity) AddTeamMember(teamId, userId string) error {
 	return ue.store.SetTeamMember(teamId, tm)
 }
 
+func (ue *UserEntity) GetTeamMembers(teamId string, page, perPage int) (error) {
+	members, resp := ue.client.GetTeamMembers(teamId, page, perPage, "")
+	if resp.Error != nil {
+		return resp.Error
+	}
+	for _, tm := range members {
+		if err := ue.store.SetTeamMember(teamId, tm); err != nil {
+			return err
+		}
+	}
```

Comment:
I think here we should implement and use a `MemStore.SetTeamMembers` (essentially identical to the existing `MemStore.SetChannelMembers` ) which will replace all elements with the ones provided. Otherwise we are just adding or updating.
For reference, same *issue* has been discussed earlier in https://github.com/mattermost/mattermost-load-test-ng/pull/16.